irma/cmd/session.go

```diff
@@ -42,42 +42,65 @@ with --message flags, or it can be specified as JSON to the --request flag.`,
 irma session --sign irma-demo.MijnOverheid.root.BSN --message message
 irma session --issue irma-demo.MijnOverheid.ageLower=yes,yes,yes,no --disclose irma-demo.MijnOverheid.root.BSN
 irma session --request '{"type":"disclosing","content":[{"label":"BSN","attributes":["irma-demo.MijnOverheid.root.BSN"]}]}'
-irma session --server http://localhost:8088 --authmethod token --key mytoken --disclose irma-demo.MijnOverheid.root.BSN`,
+irma session --server http://localhost:8088 --authmethod token --key mytoken --disclose irma-demo.MijnOverheid.root.BSN
+irma session --server http://localhost:8088 --static mystaticsession`,
 	Run: func(cmd *cobra.Command, args []string) {
-		request, irmaconfig, err := configureSession(cmd)
-		if err != nil {
-			die("", err)
+		var (
+			request    irma.RequestorRequest
+			irmaconfig *irma.Configuration
+			pkg        *server.SessionPackage
+			result     *server.SessionResult
+			err        error
+
+			flags         = cmd.Flags()
+			url, _        = flags.GetString("url")
+			serverURL, _  = flags.GetString("server")
+			noqr, _       = flags.GetBool("noqr")
+			pairing, _    = flags.GetBool("pairing")
+			authMethod, _ = flags.GetString("authmethod")
+			key, _        = flags.GetString("key")
+		)
+		if url != defaulturl && serverURL != "" {
+			die("Failed to read configuration", errors.New("--url can't be combined with --server"))
 		}
 
-		// Make sure we always run with latest configuration
-		flags := cmd.Flags()
-		disableUpdate, _ := flags.GetBool("disable-schemes-update")
-		if !disableUpdate {
-			if err = irmaconfig.UpdateSchemes(); err != nil {
-				die("failed updating schemes", err)
+		if staticFlag := flags.Lookup("static"); staticFlag.Changed {
+			if serverURL == "" {
+				die("Failed to read configuration", errors.New("--static must be combined with --server"))
+			}
+			pkg, err = staticRequest(serverURL, staticFlag.Value.String(), authMethod, key)
```

Comment:
I have to look at this again, because propably static sessions only give a `sessionPtr` instead of a full session package.